README.md

```diff
@@ -159,6 +159,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## Cloud management solutions
   * [Brainboard](https://www.brainboard.co) - Collaborative solution to visually build and manage cloud infrastructures from end-to-end.
+  * [Cloud 66](https://www.cloud66.com/) - Free for personal projects. Cloud 66 gives you everything you need to build, deploy and grow your applications on any cloud, without the headache of the “server stuff”.
```

Comment:
Please add information about the limits of this free account - 1 deployment server, 1 static site.

Reply:
Thank you - the info about the free tier includes 1 deployment server and 1 static site.